be/src/util/mysql_row_buffer.cpp

```diff
@@ -80,6 +80,25 @@ MysqlRowBuffer<is_binary_format>::MysqlRowBuffer()
           _dynamic_mode(0),
           _len_pos(0) {}
 
+template <bool is_binary_format>
+MysqlRowBuffer<is_binary_format>::MysqlRowBuffer(MysqlRowBuffer<is_binary_format>&& other) {
```

Comment:
why do we need this move ctor?

Reply:
This pull request introduced a change where a std::vector of MysqlRowBuffer was created to reuse the buffer. When the vector needs to be resized, the move constructor will be called.